src/test/regress/expected/failure_connection_establishment_0.out

```diff
@@ -0,0 +1,450 @@
+--
```

Comment:
maintaining soo many alternative files is very hard. Do we really need all of them?

Reply:
I see your point but with libpq changes, wach version has its own messages. Therefore yes it is required. I encountered all the combinations for messages. But If you think that message is not correct I can check it

Reply:
As we discussed offline, I guess Onder meant exploring whether making use of `normalize.sed` instead of those alternative outputs could work or not.

Reply:
I tried to use normalize.sed as much as possible. I could eliminate some outputs. I tried creating regex for the changes for the remaining ones however I can not create a general rule without affecting other files.

Reply:
If we want to reduce test outputs, I think we can add a regex structure which will be able to work just for one file. By this way, adding a general regex will not affect other files

Reply:
IMO, It's ok to change other files. It would be easier to maintain if all matching messages are replaced in all the files.

I think it is safer, and better to update more tests to achieve consistency

Reply:
I needed to remove the rule to replace 
``
WARNING:  connection not open
``
with 
```
WARNING:  server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
connection not open
```